src/gameplay/WaitForHost.js

```diff
@@ -18,14 +16,14 @@ function WaitForHost(props){
               console.log("Game started");
               setStart(true);
           }
-        })
+        });
         return () => subscribe();
-      }, []);
```

Comment:
why do you not need this? just curious - I thought the empty array was useful to prevent re rendering - i think I read that somewhere

Reply:
Oh wow, I didn't know that. I'm using webstorm and it was complaining about it, but you are right. I will add that back in.